contracts/upgradeable_contracts/BaseBridgeValidators.sol

```diff
@@ -30,6 +30,25 @@ contract BaseBridgeValidators is EternalStorage, Ownable {
         return (2, 1, 0);
```

Comment:
Should we increase the version here?

Reply:
Yes, Thanks! Updated here https://github.com/poanetwork/poa-bridge-contracts/pull/167/commits/7581f1241a8152fb03c6a8be3569d93005b2b664